Source/CMakeLists.txt

```diff
@@ -1,9 +1,17 @@
-cmake_minimum_required(VERSION 3.15)
+cmake_minimum_required(VERSION 3.20)
```

Comment:
Other CMakeLists.txt specify a minimum version of 3.30; is there a reason why we set it to 3.20 instead of 3.30?

Reply:
That's a great question. I think that's probably a typo, although there's nothing in this file that needs greater than 3.15.

I'd like to get the build system refactored to the point we can revert these files back to their upstream version, but that will take some doing.

Reply:
Gotcha.